precompiles/src/alpha.rs

```diff
@@ -32,14 +32,16 @@ where
             <pallet_subtensor_swap::Pallet<R> as SwapHandler<R::AccountId>>::current_alpha_price(
                 netuid.into(),
             );
-        Ok(U256::from(price.saturating_to_num::<u64>()))
+        let price_scaled = price.saturating_mul(U96F32::saturating_from_num(1_000_000_000));
+        Ok(U256::from(price_scaled.saturating_to_num::<u64>()))
```

Comment:
Could be nice to have the 1_000_000_000 set as the EVM_TO_SUBSTRATE_DECIMALS and set this constant in the runtime common instead of the main runtime file to be able to reuse it.